app/boards/shields/romac_plus/Kconfig.defconfig

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2020 Pete Johanson, Richard Jones
```

Comment:
We are trying to move towards a "single authors file" approach where we can, so if you're comfortable with it, I would ask if you wouldn't mind using headers like:

```suggestion
# Copyright (c) 2020 The ZMK Contributors
```

And then if you want to, you're welcome to add yourself to the end of the `AUTHORS` file at the top of the repo.

Reply:
Updated as a part of [4c09204](https://github.com/zmkfirmware/zmk/pull/198/commits/4c092044ce22f0c60c3bf0ef860ecc543328d5bf) and added this to romac_plus.conf for consistency.